sdk/playwrighttesting/microsoft-playwright-testing/src/reporter/mptReporter.ts

```diff
@@ -317,7 +317,7 @@ class MPTReporter implements Reporter {
         )}`;
         if (
           this.sasUri === undefined ||
-          !ReporterUtils.isTimeGreaterThanCurrentPlus10Minutes(this.sasUri.expiresAt)
+          !ReporterUtils.isTimeGreaterThanCurrentPlus10Minutes(this.sasUri.uri)
```

Comment:
Pass `this.sasUri` instead of `this.sasUri.uri`. In the future, when the service returns an appropriate expiryAt timestamp, no changes would be required here.